src/linear_algebra/tensor_algebra/grading.lean

```diff
@@ -35,14 +35,14 @@ variables {R M}
 
 /-- The tensor algebra is graded by the powers of the submodule `(tensor_algebra.ι R).range`. -/
 instance graded_algebra :
-  graded_algebra ((^) (ι R : M →ₗ[R] tensor_algebra R M).range : ℕ → submodule R _) :=
+  graded_algebra (λ i : ℕ, (ι R : M →ₗ[R] tensor_algebra R M).range ^ i) :=
```

Comment:
Arguably this spelling, while shorter, is worse because it produces an un-delta-reduced type. It probably doesn't matter though.

Reply:
Okay, let me change it back. The shortest spelling in delta-reduced form is probably `  graded_algebra ((^) (ι R : M →ₗ[R] _).range : ℕ → submodule R _)`, but writing out the `tensor_algebra R M` once is clearer.